great_expectations/expectations/expectation.py

```diff
@@ -1380,6 +1381,14 @@ def run_diagnostics(
             if test_result.error_diagnostics
         ]
 
+        # If run for the gallery, don't include a bunch of stuff
+        if for_gallery:
+            examples = []
+            gallery_examples = []
```

Comment:
Why don't we want these things for the gallery?

Reply:
They are not being used by the gallery frontend at all anymore as of earlier this month. They weren't that useful.

One of the things we are going to do is come up with more real examples to put in the docstrings like https://deploy-preview-198--gxbrandupdate.netlify.app/expectations/expect_column_distinct_values_to_be_in_set

Reply:
@kenwade4 Do we still need to run the tests in the gallery if we are throwing away the test_results and errors? Would probably speed up the build if we don't.

Reply:
Yea we need to run them all still because the summary info is included in the `backend_test_result_counts` key

```
        "tests": [],
        "backend_test_result_counts": [
            {
                "backend": "pandas",
                "num_passed": 6,
                "num_failed": 0,
                "failing_names": null
            },
            {
                "backend": "sqlite",
                "num_passed": 6,
                "num_failed": 0,
                "failing_names": null
            }
```

Reply:
Those power the logic behind the green, yellow, and red squares in the "datasource view" of the gallery.

If num_passed >= 1 and num_failed == 0 then green box
if num_passed ==0 and num_failed >= 1 then red box
if num_passed >=1 and num_failed >= 1 then yellow box
if num_passed == 0 and num_failed == 0 then grey empty box